src/components/BulkEnrollmentPage/CourseSearchResults.test.jsx

```diff
@@ -41,8 +43,17 @@ const searchResults = {
       key: 'foo',
       partners: [{ name: 'edX' }, { name: 'another_unused' }],
     },
+    {
+      title: testCourseName2,
+      advertised_course_run: {
+        key: testCourseRunKey2,
+        start: testStartDate2,
+      },
+      key: 'foo2',
+      partners: [{ name: 'edX' }, { name: 'another_unused' }],
+    },
   ],
-  page: 3,
+  page: 0,
```

Comment:
The Algolia results made no sense to me and were also making it impossible to test multiple results on the same page.